com.blockstream.Green.yml

```diff
@@ -0,0 +1,39 @@
+app-id: com.blockstream.Green
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: green
+modules: 
+  - name: green
+    buildsystem: simple
+    build-commands:
+      - bsdtar -xf BlockstreamGreen_Linux_x86_64.tar.gz
+      - install -D green ${FLATPAK_DEST}/bin/green
+      - desktop-file-install --dir ${FLATPAK_DEST}/share/applications com.blockstream.Green.desktop
+      - install -Dm644 green.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.blockstream.Green.png
+      - install -Dm644 com.blockstream.Green.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.blockstream.Green.metainfo.xml
+
+    sources:
+      - type: file
```

Comment:
you should use the archive type instead and let flatpak extract the archive.

Reply:
Ok removed the `bsdtar` command.